locales/el-GR.json

```diff
@@ -0,0 +1,746 @@
+      "description": "Σίγουρα θες να αποκλείσεις τον {0}?;",
+      "title": "Αποκλεισμός τομέα"
+    },
+    "common": {
+      "cancel": "Όχι",
+      "confirm": "Ναι"
+    },
+    "delete_list": {
+      "cancel": "Ακύρωση",
+      "confirm": "Διαγραφή",
+      "description": "Θες σίγουρα να διαγράψεις την λίστα \"{0}\";",
+      "title": "Διαγραφή λίστας"
+    },
+    "delete_posts": {
+      "cancel": "Ακύρωση",
+      "confirm": "Διαγραφή",
+      "description": "Θες σίγουρα να διαγράψεις αυτή την ανάρτηση;",
+      "title": "Διαγραφή ανάρτησης"
+    },
+    "mute_account": {
+      "cancel": "Ακύρωση",
+      "confirm": "Σίγαση",
+      "days": "μέρες|μέρα|μέρες",
+      "description": "Θες σίγουρα να κάνεις σίγαση στον {0};",
+      "hours": "ώρες|ώρα|ώρες",
+      "minute": "λεπτά|λεπτό|λεπτά",
+      "notifications": "Σίγαση ειδοποιήσεων",
+      "specify_duration": "Προσδιόρισε διάρκεια σίγασης",
+      "title": "Σίγαση λογαριασμού"
+    },
+    "show_reblogs": {
+      "cancel": "Ακύρωση",
+      "confirm": "Εμφάνιση",
+      "description": "Θες σίγουρα να εμφανίζεις τις ενισχύσεις του {0}?",
+      "title": "Εμφάνιση ενισχύσεων"
+    },
+    "unfollow": {
+      "cancel": "Ακύρωση",
+      "confirm": "Άρση ακολούθησης",
+      "description": "Θες σίγουρα να κάνεις άρση ακολούθησης στον {0};",
+      "title": "Άρση ακολούθησης"
+    }
+  },
+  "conversation": {
+    "with": "με"
+  },
+  "custom_cards": {
+    "stackblitz": {
+      "lines": "Γραμμές {0}",
+      "open": "Άνοιγμα",
+      "snippet_from": "Απόκομμα από {0}"
+    }
+  },
+  "error": {
+    "account_not_found": "Ο λογαριασμός {0} δεν βρέθηκε",
+    "explore_list_empty": "Δεν υπάρχει τίποτα δημοφιλές αυτήν τη στιγμή. Έλεγξε ξανά αργότερα!",
+    "file_size_cannot_exceed_n_mb": "Το μέγεθος του αρχείου δεν μπορεί να υπερβαίνει τα {0}MB",
+    "sign_in_error": "Σφάλμα σύνδεσης με τον διακομιστή",
+    "status_not_found": "Η ανάρτηση δεν βρέθηκε",
+    "unsupported_file_format": "Μη υποστηριζόμενη μορφή αρχείου"
+  },
+  "help": {
+    "build_preview": {
+      "desc1": "Βλέπεις, για την ώρα, μια δοκιμαστική έκδοση του Elk από την κοινότητα - {0}.",
+      "desc2": "Μπορεί να περιέχει μη αξιολογημένες ή ακόμα και κακόβουλες αλλαγές.",
+      "desc3": "Μην συνδεθείς με τον πραγματικό σου λογαριασμό",
+      "title": "Εκκίνηση προεπισκόπησης"
+    },
+    "desc_highlight": "Αναμένονται μερικά σφάλματα και ελλείψεις δυνατοτήτων πού και πού.",
+    "desc_para1": "Ευχαριστούμε για το ενδιαφέρον σου να δοκιμάσεις το Elk, τον εν εξελίξει πελάτη Mastodon για τον ιστό!",
+    "desc_para2": "Δουλεύουμε σκληρά στην ανάπτυξη και βελτίωσή του με τον καιρό.",
+    "desc_para3": "To boost development, you can sponsor the Team through GitHub Sponsors. We hope you enjoy Elk!",
+    "desc_para4": "Elk is Open Source. If you'd like to help with testing, giving feedback, or contributing,",
+    "desc_para5": "reach out to us on GitHub",
+    "desc_para6": "and get involved.",
```

Comment:
@trlef19 Thank you for all your effort on these translations! 🙂 Could you also translate these a few remaining strings? Thanks.